webhooks-extension/config/release/gcr-tekton-webhooks-extension.yaml

```diff
@@ -161,15 +161,22 @@ spec:
       - name: pull-request
         type: pullRequest
     params:
-      - name: pipelinerun
-        description: The name of the pipelinerun to be monitored
+      - name: pipelineruns
+        description: The name of the pipelineruns (and their namespace) to be monitored in format pipelinerun1:namespace1, pipelinerun2:namespace2
         default: default pipelinerun
       - name: commentsuccess
         description: The text of the success comment
-        default: Pipeline Succeeded
+        default: "Success"
       - name: commentfailure
         description: The text of the failure comment
-        default: Pipeline Failed
+        default: "Failed"
+      - name: dashboard-url
+        description: The URL to the pipelineruns page of the dashboard
+        default: "http://localhost:9097/"
+      # This can be deleted after pending status change
+      - name: secret
+        description: The secret containing the access token to access github
+      # Up to here
```

Comment:
Same as above

Reply:
ignoring.... this marks the end of a snippet that can be deleted